random.c

```diff
@@ -140,7 +140,7 @@ int getccol(int bflg)
 			col |= tabmask;
 		else if (c < 0x20 || c == 0x7F)
 			++col;
-		else if (c >= 0xc0 && c <= 0xa0)
```

Comment:
It should be c >= 0x80 && c <= 0xa0, see 'vtputc()' function in display.c.

Reply:
I agree. Intent is clearer in commit ddd45dbf file basic.c function `getgoal()` line 44.